### ISSUE TITLE
React / Project 3 - CSS Bug

### ISSUE THREAD

**Issue by SamRTheDev** [SamPuckett/50Projects50Days#11]
There's currently a bug with the css in the React portion of the project where if the pink nav button is clicked, the pink circle shifts down around 50px and I couldn't figure out where in the css those style rules were being added.

**Comment by itsdell** [comment]
Hello, can I help out with this bug?

**Comment by SamPuckett** [comment]
> Hello, can I help out with this bug?

Of course, if you'd like to, go ahead! I believe it has to do with the CSS inside of the 'App.css' or in the CSS files for the actual components.
